source/pattern-library/cards/utilization-trend-card/index.md

```diff
@@ -93,3 +95,4 @@ layout: page
     [ usability content goes here ]
   </div>
 </div>
+<script src="/ngdocs/formatForPFsite.js"></script>
```

Comment:
Additional scripts on a per page basis can be added via front matter.  See line 4 of https://raw.githubusercontent.com/patternfly/patternfly-toolkit/master/source/index.md for an example of how to do so.